### ISSUE TITLE
All pages: list update after element removal is always removing the last element from the list

### ISSUE THREAD

**Issue by manrodrigues** [ns1labs/orb#1052]
When some element is removed from orb, although the deletion is being performed on the correct element, the list of all elements is removing always the last element and enabling the user to "remove" multiple times the same element.
After refresh the page, elements are displayed correct.

Agents:

![agent_removal_bug.gif](https://images.zenhubusercontent.com/61aa14a937730b887dff82b3/c2222dac-9fa6-41b7-bd40-e836b8ca60b6)

Groups:

![group_removal_bug.gif](https://images.zenhubusercontent.com/61aa14a937730b887dff82b3/87b37d72-132d-4ef8-b2c8-18ff30ef48bb)

Policies:

![policy_removal_bug.gif](https://images.zenhubusercontent.com/61aa14a937730b887dff82b3/442a49c4-fdb5-49c2-82c0-842210a7219f)

Steps to reproduce:
- Create multiple elements on orb (for example, 5 agents)
- Delete one of the elements from orb (choose anyone other than the last on the list)
- Check the list updated

Expected result:
- Removed element should be removed from the updated list

Current result:
- The last element is always being displayed as removed (is not removed in fact)

**Comment by gpazuch** [comment]
@manrodrigues could not verify this is the case.

it indeed excludes one element, it takes little time for the table to refresh and redraw.

**Comment by manrodrigues** [comment]
@gpazuch the problem is that if I exclude an element that are on the top positions of the list, there are no reason to the the last element from the table being remove (even for a while).. If table takes some time to refresh, not should happen on this time.. remove the last element it's not the expected behavior.

**Comment by gpazuch** [comment]
![list-removal](https://user-images.githubusercontent.com/1490938/166458994-1b22bae6-8b34-4fb3-9eb5-4d2217b43fe8.gif)
@manrodrigues I didn't quite see the effect.

**Comment by manrodrigues** [comment]
@gpazuch I agree that the priority was not accurate.. i changed it.. but if you consider a page with less than 10 itens (all of then listed on the first page) you will can be able to see the effect, so, I continue thinking that we should solve it when we can.

![removal_last_element](https://user-images.githubusercontent.com/78241475/166478543-0792481a-79eb-422e-8b02-1acadc3180d0.gif)
